test/extended/router/gatewayapicontroller.go

```diff
@@ -246,6 +247,44 @@ var _ = g.Describe("[sig-network-edge][OCPFeatureGate:GatewayAPIController][Feat
 		g.By("Validating the http connectivity to the backend application")
 		assertHttpRouteConnection(defaultRoutename)
 	})
+
+	g.It("Ensure GIE is enabled after creating an inferencePool CRD", func() {
+		errCheck := checkGatewayClass(oc, gatewayClassName)
+		o.Expect(errCheck).NotTo(o.HaveOccurred(), "GatewayClass %q was not installed and accepted", gatewayClassName)
+
+		g.By("Install the GIE CRD")
+		err := oc.AsAdmin().Run("create").Args("-f", infPoolCRD).Execute()
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		g.By("Confirm istio is healthy and contains the env variable")
+		waitForIstioHealthy(oc)
+		waitIstioErr := wait.PollUntilContextTimeout(context.Background(), 1*time.Second, 5*time.Minute, false, func(context context.Context) (bool, error) {
+			istioEnv, err := oc.AsAdmin().Run("get").Args("-n", "openshift-ingress", "istio", "openshift-gateway", "-o=jsonpath={.spec.values.pilot.env}").Output()
+			o.Expect(err).NotTo(o.HaveOccurred())
+			if strings.Contains(istioEnv, `ENABLE_GATEWAY_API_INFERENCE_EXTENSION":"true`) {
```

Comment:
The quotes on the substring to be searched looks a little strange. Should it be `"ENABLE_GATEWAY_API_INFERENCE_EXTENSION":"true"` just to have complete quoting?